app/controllers/admins_controller.rb

```diff
@@ -113,7 +113,7 @@ def verify_params
   #
   # This is a temporary `verify_params` method that will exist until we migrate fully to the new permissions system
   #
-  def 🆕verify_params
```

Comment:
please, no emoji in code 😁 .  I know editors and ruby can handle it, but it makes grepping or finding things harder.